src/classes/ACCT_Accounts_TDTM.cls

```diff
@@ -39,8 +39,8 @@
 * name updating.
 */
 public class ACCT_Accounts_TDTM extends TDTM_Runnable {
-        
```

Comment:
No logic change in this class. Adv does not use NPSP's HouseholdNaming logic, so I extract the rest logic into a separate function. So, adv can call it.